Added support for a fully loaded, scrollable, contained ladder.
If you want to disable it, reload the page or whatever.

LynnCinnamon opened the pull request.

LynnCinnamon committed:
Added support for a fully loaded, scrollable, contained ladder.
This setting can be enabled (but not disabled because I cant be bothered to implement this) via a button in the settings menu.
If you want to disable it, reload the page or whatever.

LynnCinnamon committed:
Added "expandLadder" to global space

Quadrum1 commented on fairgame.js:
Can you add an option to call expandLadder() automatically?
if (qolOptions.scrollableLadder) {expandLadder()}

Quadrum1 commented on fairgame.js:
Would need to add that config at the top as well then

Quadrum1 commented on fairgame.js:
This could lead to some client lag, better keep it at 250-500 or so?

Quadrum1 commented on fairgame.js:
Having two buttons with id "rowsButton" could lead to issues.

LynnCinnamon commented on fairgame.js:
250 -500 would not give us the entire ladder.
It should only lag if we have actually that many rankers. I will add an option for the input size.

LynnCinnamon committed:
Implemented Changes requested by Quandrum1
+ option to call expandLadder() automatically
+ config of call expandLadder() automatically
+ standard size of ladder set to 250
+ changed button id

Quadrum1 commented on fairgame.js:
1 = 1
:smile:
